.gitlab-ci.yml

```diff
@@ -91,6 +98,126 @@ xliff-lint:
     - xmllint --schema ./xliff-core-1.2-strict.xsd
       --noout $(find Resources -name '*.xlf')
 
+unit-tests-php7.2-t3-v10:
+  image: php:7.2
+  stage: test
+  needs:
+    - build-composer-dependencies
+  script:
+    - composer ci:tests:unit
+
+unit-tests-php7.3-t3-v10:
+  image: php:7.3
+  stage: test
+  needs:
```

Comment:
This also should depend on the PHP 7.3 linting. (If the PHP is not syntactically valid, there is no point in running the test.)